downloads/az/github-git-cheat-sheet.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: cheat-sheet
+redirect_to: false
+title: GitHub-dan Git üçün yaddaş kitabçası
+byline: Git - kompüterinizdə GitHub ilə işləmək üçün istifadə olunan açıq mənbəli, versiyaların paylanmış nəzarət sistemidir. Bu kitabça, əmr sətri üçün əsas Git əmrləri və onların tətbiqi üzrə təlimatlardan ibarətdir.
+leadingpath: ../../
+---
+
+{% capture colOne %}
+## Git quraşdırılması
+GitHub, repozitarla əsas əməliyyatların icra edilməsi üçün qrafik interfeysli bir pəncərə proqramı və genişləndirilmiş iş ssenariləri üçün Git-in avtomatik yenilənən konsol versiyasını təqdim edir.
+
+### Masaüstü GitHub
+[desktop.github.com](https://desktop.github.com)
+
+Linux və POSIX sistemləri üçün Git distributivlərini Git SCM rəsmi saytında tapa bilərsiniz.
+
+### Bütün platformalar üçün Git
+[git-scm.com](http://git-scm.com)
+
+## İlkin quraşdırma
+İstifadəçi haqqında informasiyanın bütün lokal repozitarlar üçün konfiqurasiyası
+
+```$ git config --global user.name "[istifadəçi_adınız]"```
```

Comment:
Remove underscore from "istifadəçi_adınız" to match it with [e-poçt ünvanınız]: `"istifadəçi adınız"`. Also, I do not think you need to fere to these values as you are referring to the reader. So, instead of `"istifadəçi adınız"`, use `"istifadəçi adı"` and instead of `"[e-poçt ünvanınız]"` use `"[e-poçt ünvanı]"`.